api/v1alpha1/networkchaos_types.go

```diff
@@ -369,6 +378,43 @@ func (corrupt *CorruptSpec) ToNetem() (*chaosdaemonpb.Netem, error) {
 	}, nil
 }
 
+// BandwidthSpec defines details of bandwidth limit.
+type BandwidthSpec struct {
+	// Rate is the speed knob. Allows bps, kbps, mbps, gbps, tbps unit. bps means bytes per second.
+	Rate string `json:"rate"`
+	// Limit is the number of bytes that can be queued waiting for tokens to become available.
+	Limit uint32 `json:"limit"`
+	// Buffer is the maximum amount of bytes that tokens can be available for instantaneously.
+	Buffer uint32 `json:"buffer"`
```

Comment:
What about `peakrate` and `minburst`?

Should we set some default values for them? (Have you tested what will happen if we set some of these fields 0? Will this 0 means no limit or zero limit?)